modules/firstPartyData/index.js

```diff
@@ -0,0 +1,269 @@
+/**
+ * This module sets default values and validates ortb2 first part data
+ * @module modules/firstPartyData
+ */
+
+import { config } from '../../src/config.js';
+import * as utils from '../../src/utils.js';
+import { getHook } from '../../src/hook.js';
+import { getRefererInfo } from '../../src/refererDetection.js'
+import { getStorageManager } from '../../src/storageManager.js';
+
+const STORAGE = getStorageManager();
+let ortb2 = {};
+let globalConfig = {};
+let win = (window === window.top) ? window : window.top;
+
+/**
+ * Checks for referer and if exists merges into ortb2 global data
+ */
+function setReferer() {
+  if (getRefererInfo().referer) utils.mergeDeep(ortb2, {site: {ref: getRefererInfo().referer}});
```

Comment:
Correct. So left to right in terms of priority. This line is actually just setting the modules global variable which is later merged with the response of getConfig('ortb2) || {}. So if the value exists in ortb2 already it will overwrite the modules 'enrichments' setting and use what was already there.